vmb/matrimony/models/profiles.py

```diff
@@ -544,6 +544,12 @@ def matching_profiles(self):
                 matches.append((m.id, m.male, m.female_response))
         return matches
 
+    def mentor(self):
+        return Mentor.objects.filter(profile=self)[0]
+
+    def expectations(self):
+        return Expectation.objects.get(self)
+
```

Comment:
Same goes for here, as well. Make method `property`. And the query should be updated to 

```
self.expectations.all()[0]
```

See, `related_name` here: https://github.com/jpsoffice/vmb/blob/master/vmb/matrimony/models/profiles.py#L655